seedfarmer/services/_codebuild.py

```diff
@@ -138,6 +142,26 @@ def start(
     str
         The CodeBuild Build/Exectuion Id
     """
+    # If no dumper is provided, use PyYAML's safe_dump
```

Comment:
Why different yaml dumpers are needed? We were fine with just using `yaml.safe_dump`

Reply:
The formatting of multiline commands and properly formatting them were not being recognized by the bash script when converting from a python list...the multi-line was being corrupted